### PR TITLE
[Notifier] Add missing deprecation entry

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       |  
| License       | MIT
| Doc PR        | 

follow-up on #41298

### CODE REVIEW

File: UPGRADE-5.2.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Notifier
  * [BC BREAK] Changed the type-hint of the `$recipient` argument in `NotifierInterface::send()`,
    `Notifier::getChannels()`, `ChannelInterface::notifiy()` and `ChannelInterface::supports()` to
    `RecipientInterface`.
+ * symfony/slack-notifier: Deprecated `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```
Comment (jschaedl) [inline UPGRADE-5.2.md]:
```suggestion
 * symfony/slack-notifier: Deprecate `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```

File: UPGRADE-5.2.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ Notifier
  * [BC BREAK] Changed the type-hint of the `$recipient` argument in `NotifierInterface::send()`,
    `Notifier::getChannels()`, `ChannelInterface::notifiy()` and `ChannelInterface::supports()` to
    `RecipientInterface`.
+ * symfony/slack-notifier: Deprecate `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```
Comment (OskarStark) [inline UPGRADE-5.2.md]: I would also add an UPGRADE file to the bridge itself, because it is a separate package.

WDYT @derrabus ?

Comment (jschaedl) [reply UPGRADE-5.2.md]: I think this is a good idea.

Comment (wouterj) [reply UPGRADE-5.2.md]: Symfony only has seperate CHANGELOG files. The UPGRADE file is shared for all subtree splits in the root

Comment (OskarStark) [reply UPGRADE-5.2.md]: Ok is the prefix with the package name fine? Or should it be a subheading?

Comment (derrabus) [reply UPGRADE-5.2.md]: I would just remove the prefix, tbh.

```suggestion
 * Deprecate `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```

Apart from that, the deprecation was added in 5.1, so the entry should be moved to `UPGRADE-5.1.md`.

Comment (jschaedl) [reply UPGRADE-5.2.md]: > Apart from that, the deprecation was added in 5.1, so the entry should be moved to UPGRADE-5.1.md.

🙈  You are right, I mixed it up.

File: UPGRADE-5.2.md [hunk 3]
```diff
@@ -173,4 +173,3 @@ Security
  * Deprecated the `AbstractRememberMeServices::$providerKey` property in favor of
    `AbstractRememberMeServices::$firewallName`, the old property will be removed
    in 6.0.
-
```
Comment (derrabus) [inline UPGRADE-5.2.md:176]: Should be reverted.

File: UPGRADE-6.0.md [hunk 4]
```diff
@@ -113,6 +113,11 @@ Monolog
  * The `$actionLevel` constructor argument of `Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy` has been replaced by the `$inner` one which expects an ActivationStrategyInterface to decorate instead. `Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy` is now final.
  * The `$actionLevel` constructor argument of `Symfony\Bridge\Monolog\Handler\FingersCrossed\HttpCodeActivationStrategy` has been replaced by the `$inner` one which expects an ActivationStrategyInterface to decorate instead. `Symfony\Bridge\Monolog\Handler\FingersCrossed\HttpCodeActivationStrategy` is now final.
 
+Notifier
+--------
+
+ * symfony/slack-notifier: Remove `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```
Comment (derrabus) [inline UPGRADE-6.0.md]:
```suggestion
 * Remove `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```

File: UPGRADE-5.2.md (hunk 3 above)
Comment (jschaedl) [reply UPGRADE-5.2.md:176]: 🤔  There were two lines at the end of the file. That's why I removed one:

Screenshot 2021-05-20 at 16 21 42

**Review by derrabus — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @jschaedl.

**Pull request merged**